src/Presentation/Nop.Web/Areas/Admin/Factories/ICommonModelFactory.cs

```diff
@@ -110,4 +111,14 @@ public partial interface ICommonModelFactory
     /// <param name="models">List of system warning models</param>
     /// <returns>A task that represents the asynchronous operation</returns>
     Task PreparePluginsWarningModelAsync(IList<SystemWarningModel> models);
+
+    /// <summary>
+    /// Prepare multistore preview models
+    /// </summary>
+    /// <param name="model">An admin BaseNopEntityModel</param>
+    /// <returns>
+    /// A task that represents the asynchronous operation
+    /// The task result contains the list of multistore preview models
+    /// </returns>
+    Task<IList<MultistorePreviewModel>> PrepareMultistorePreviewModelsAsync(object model);
```

Comment:
The `CommonModelFactory` felt like the right place to put this as it is common across a subset of pages.
Otherwise it could go into its own Factory :)